Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ruby:2.7.0-slim
+ARG precompileassets
+
+RUN apt-get update && apt-get install -y curl gnupg
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+RUN curl -q https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+RUN apt-get -y update && \
+      apt-get install --fix-missing --no-install-recommends -qq -y \
+        build-essential \
+        vim \
+        wget gnupg \
+        git-all \
+        curl \
+        ssh \
+        postgresql-client-10 libpq5 libpq-dev -y && \
+      wget -qO- https://deb.nodesource.com/setup_9.x  | bash - && \
+      apt-get install -y nodejs && \
+      wget -qO- https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+      echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+      apt-get update && \
+      apt-get install yarn && \
+      apt-get clean && \
+      rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN gem install bundler
+#Install gems
+RUN mkdir /gems
+WORKDIR /gems
+COPY Gemfile .
+COPY Gemfile.lock .
+RUN bundle install
+
+ARG INSTALL_PATH=/opt/elderly
```

Comment:
path is `elderly`?

Reply:
Inside the container the app lives in /opt/elderly

Reply:
inside the container the app lives in /opt/eldery